src/main/groovy/org/pih/warehouse/inventory/CycleCountDto.groovy

```diff
@@ -0,0 +1,20 @@
+package org.pih.warehouse.inventory
+
+class CycleCountDto {
+
+    String id
+
+    String status
+
+    List<CycleCountItemBasicDto> cycleCountItems
+
+    static CycleCountDto createFromCycleCountItems(List<CycleCountItem> items) {
```

Comment:
I gotta say it was quite tricky to work with the items without having the bidirectional relationship between Cycle count and Cycle count items, but I remember the reasons for promoting the unidirectional one-to-many relationship.